packages/dx-react-scheduler/docs/guides/views.md

```diff
@@ -0,0 +1,33 @@
+# React Scheduler - Views
+
+The Scheduler component provides several types of views. Views display information in various ways, you can choose the most convenient view for your data source or combine few views and switching between them.
+
+## Related Plugins
+
+The following plugins render corresponding views:
+
+- [DayView](../reference/day-view.md) - renders a day layout
+- [WeekView](../reference/week-view.md) - renders a week layout
+- [MonthView](../reference/month-view.md) - renders a month layout
+
+## DayView
+
+The `DayView` displays appointments for a specified date. The Scheduler component arranges appointments from top to bottom. If their time intervals overlap, their width is decreased and they are placed next to each other.
```

Comment:
We can add here that a Day view is the most detailed. And the opposite in Month view section